tgui/packages/tgui/interfaces/BluespaceArtillery.js

```diff
@@ -1,64 +1,83 @@
 import { useBackend } from '../backend';
-import { Box, Button, LabeledList, NoticeBox, ProgressBar, Section } from '../components';
+import { Box, Button, LabeledList, NoticeBox, ProgressBar, Section, Stack } from '../components';
 import { Window } from '../layouts';
 
 export const BluespaceArtillery = (props, context) => {
   const { act, data } = useBackend(context);
-  const { notice, connected, unlocked, target, charge, max_charge } = data;
+  const { notice, connected, unlocked, target, charge, max_charge, formatted_charge, targets } = data;
```

Comment:
```suggestion
  const { notice, connected, unlocked, target_ref, target_name, charge, max_charge, formatted_charge, targets } = data;
```
Since it no longer sends `list()` and I asked you to use different two keys, it should be so.

Reply:
done